src/main/scala/mesosphere/marathon/core/matcher/manager/impl/OfferMatcherManagerActor.scala

```diff
@@ -110,10 +110,20 @@ private[impl] class OfferMatcherManagerActor private (
 
   var timerTick: Option[Cancellable] = None
 
+  var addToUnprocessedOffers: (List[UnprocessedOffer], UnprocessedOffer) => List[UnprocessedOffer] = (offersQueue: List[UnprocessedOffer], unprocessedOffer: UnprocessedOffer) =>
+    offersQueue.::(unprocessedOffer)
   override def preStart(): Unit = {
     implicit val ec = context.system.dispatcher
     val timeout = conf.offerMatchingTimeout()
     timerTick = Some(context.system.scheduler.schedule(timeout, timeout, self, CleanUpOverdueOffers))
+    // choose the strategy of insertion at initialization for the unprocessed offers queue
+    if (conf.queuedOffersFifo.toOption.get == true) {
```

Comment:
do we need to test equality to true or could we simply write

if (conf.queuedOffersFifo.toOption.get)

also if it's an option type, how will this code react if there is no value?

Reply:
The option has a default value so when the flag is not activated the option would have Some(false) as value. So I think there is no way that this Option wouldn't have a value

Reply:
ok